src/session/Manager.cc

```diff
@@ -75,7 +75,25 @@ class ProtocolStats {
 
 } // namespace detail
 
-Manager::Manager() { stats = new detail::ProtocolStats(); }
+Manager::Manager() {
+    stats = new detail::ProtocolStats();
+    killed_by_inactivity_metric =
+        telemetry_mgr->CounterInstance("zeek", "sessions_killed_by_inactivity", {},
```

Comment:
It would be nice to name that one such that it ends in "sessions_total", so `zeek_killed_by_inactivity_sessions_total`, or move to the labeled variant for the future>